### DISCUSSION TITLE
Bticino L4531C e Zigbee2Mqtt - problema accoppiamento (join) device

### DISCUSSION THREAD

**Discussion by evil-life** [indomus/forum#2729] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti,
ho un'installazione di Zigbee2Mqtt su Proxmox (container separati di z2m, mqtt e home assistant) con chiavetta Sonoff dongle P.
Ho acquistato 3 dispositivi Bticino L4531C e appena arrivati ho provato ad accoppiare il primo e dopo vari reset sono riuscito nell'impresa.
Oggi ho provato ad accoppiare gli altri 2 ma non c'è verso. Premo il tasto reset per 8 secondi fino a che il led rosso si spegne e quando rilascio fa il classico "clack" del relè che dovrebbe certificare il reset.
Ma Z2M non trova i dispositivi. Ne l'uno ne l'altro. Ho provato decine di volte.
Qualcuno con lo stesso dispositivo conosce una procedura più solida di accoppiamento? Magari mi sto perdendo qualcosa, ma anche cercando in rete non trovo nulla. Sulla pagina ufficiale del dispositivo su Z2M dice che dopo qualche reset dovrebbe vederli...
Grazie a tutti

**Comment by evil-life** [comment]
OK, posto un aggiornamento che magari sarà utile a qualcuno in futuro.
Ho restituito i 2 dispositivi ad Amazon e ne ho comprati altri 2. I 2 arrivati si sono accoppiati istantaneamente appena messi in corrente.
L'unica differenza "visibile" è che i nuovi sono Made in Tunisia, mentre gli altri 2 che ho restituito erano Made in Italy.
La mia teoria è che su quelli restituiti ci fosse un firmware diverso non compatibile con Z2M (o magari una vecchia versione che andrebbe quindi aggiornata attraverso l'hub e l'app proprietari).

**Reply by xraver** [reply]
Vedo che ci sono diversi problemi ultimamente tra i dispositivi. io ho comprato dimmer e tapparella e funzionano perfettamente, deviatore e relè no. pare ci sia un aggiornamento che dovrebbe risolvere in arrivo.
